pandas/_testing/_warnings.py

```diff
@@ -155,14 +157,21 @@ def _assert_caught_no_extra_warnings(
                 #  suppress these, xref GH#38630
                 continue
 
-            extra_warnings.append(
-                (
-                    actual_warning.category.__name__,
-                    actual_warning.message,
-                    actual_warning.filename,
-                    actual_warning.lineno,
-                )
-            )
+            warning_data = [
+                actual_warning.category.__name__,
+                actual_warning.message,
+                actual_warning.filename,
+                actual_warning.lineno,
+            ]
+
+            if actual_warning.category == ResourceWarning:
```

Comment:
GH ref pointing back here?

Reply:
Don't think we have a current Github issue related to this one. Hoping that this code is just temporary to debug the CI